MLCardForm.podspec

```diff
@@ -7,8 +7,8 @@ Pod::Spec.new do |s|
   s.author           = "PX Team - Juan Sanzone - Eric Ertl - Esteban Boffa"
   s.source           = { :git => "https://github.com/mercadolibre/card-form-ios", :tag => s.version.to_s }
   s.swift_version    = '5.0'
-  s.ios.deployment_target = '10.0'
-  s.platform         = :ios, '10.0'
+  s.ios.deployment_target = '12.0'
+  s.platform         = :ios, '12.0'
```

Comment:
Why did the minimum target change?

Reply:
I'm going to remove it, I think it was when I mistakenly integrated another library that didn't compile for me
Done!